### PR TITLE
Send proper times when executing a cell

### PR DESCRIPTION
**Pull request by DonJayamanne**

### CODE REVIEW

File: src/client/datascience/jupyter/kernels/cellExecution.ts [hunk 1]
```diff
@@ -441,9 +442,9 @@ export class CellExecution implements IDisposable {
         const props = { notebook: true };
         if (!CellExecution.sentExecuteCellTelemetry) {
             CellExecution.sentExecuteCellTelemetry = true;
-            sendTelemetryEvent(Telemetry.ExecuteCellPerceivedCold, this.stopWatch.elapsedTime, props);
```
Comment (DonJayamanne) [inline src/client/datascience/jupyter/kernels/cellExecution.ts:444]: Using the `stopWatch` property is wrong.
This value gets reset just when the cell is sent to Jupyter. Thus this isn't the perceived time
E.g. when usre opens a notebook an hits run, and kernel takes 10s to start, afer it starts we send this cell to Jupyter and re-set the stopwatch, but now we're sending <10 for execution times.

Hence using a new property to capture the perceived times.

**Comment by vibhavagarwal5** [comment]
Is this change published, I see that the cell still starts from 40s or something and once it completes then only shows the correct time..

**Comment by vibhavagarwal5** [comment]
@DonJayamanne

**Comment by rchiodo** [comment]
@vibhavagarwal5 this change has nothing to do with the display of the cell execution time. It's for telemetry. Please log an issue if you are not seeing the expected time in a cell. Thanks.

**Pull request merged**